cypress/integration/simulator/simulator.spec.ts

```diff
@@ -0,0 +1,62 @@
+describe("Flow", () => {
+  beforeEach(function () {
+    // login before each
+    cy.login();
+    cy.visit("/chat");
+  });
+
+  it("should start simulator", () => {
+    cy.get('[data-testid="beneficiaryName"]').then((body) => {
+      if (body[0].innerText !== "Simulator") {
+        cy.get('[data-testid="simulatorIcon"]').click();
+        cy.get("#simulator");
+      }
+    });
+  });
+  it("should trigger help flow", () => {
+    cy.get('[data-testid="beneficiaryName"]').then((body) => {
+      if (body[0].innerText !== "Simulator") {
+        cy.get('[data-testid="simulatorIcon"]').click();
+        cy.get("#simulator");
+      }
+    });
+    const old_msg_count = cy
+      .get('*[class^="Simulator_ReceivedMessage"]')
+      .its("length");
+    cy.get("[data-testid=simulatorInput]")
+      .click()
+      .wait(500)

```

Comment:
Do we need to wait after the simulatorInput click? it's working fine without a wait also, can you please review.